spaceship/spec/base_spec.rb

```diff
@@ -1,4 +1,5 @@
 describe Spaceship::Base do
+  include_examples "common spaceship login", true
```

Comment:
I am not familiar with `include_examples` but why not change what `Spaceship::Tunes.login` does in test so that we always choose the SIRP login?

Reply:
We do always choose the SIRP login! This just adds some mocking that is _kind of_ required for SIRP because there is a `SecureRandom.hex(256)` inside of SIRP implementation when its makes the `a`.  So the `include_example` is a way to do that in each of the tests with only one line 🤷‍♂️